services/horizon/main.go

```diff
@@ -205,22 +213,26 @@ func initConfig() {
 	}
 
 	config = horizon.Config{
-		DatabaseURL:            viper.GetString("db-url"),
-		StellarCoreDatabaseURL: viper.GetString("stellar-core-db-url"),
-		StellarCoreURL:         viper.GetString("stellar-core-url"),
-		Port:                   viper.GetInt("port"),
-		RateLimit:              throttled.PerHour(viper.GetInt("per-hour-rate-limit")),
-		RedisURL:               viper.GetString("redis-url"),
-		FriendbotURL:           viper.GetString("friendbot-url"),
-		LogLevel:               ll,
-		SentryDSN:              viper.GetString("sentry-dsn"),
-		LogglyToken:            viper.GetString("loggly-token"),
-		LogglyHost:             viper.GetString("loggly-host"),
-		TLSCert:                cert,
-		TLSKey:                 key,
-		Ingest:                 viper.GetBool("ingest"),
-		HistoryRetentionCount:  uint(viper.GetInt("history-retention-count")),
-		StaleThreshold:         uint(viper.GetInt("history-stale-threshold")),
-		SkipCursorUpdate:       viper.GetBool("skip-cursor-update"),
+		DatabaseURL:                 viper.GetString("db-url"),
+		StellarCoreDatabaseURL:      viper.GetString("stellar-core-db-url"),
+		StellarCoreURL:              viper.GetString("stellar-core-url"),
+		Port:                        viper.GetInt("port"),
+		RateLimit:                   throttled.PerHour(viper.GetInt("per-hour-rate-limit")),
+		RedisURL:                    viper.GetString("redis-url"),
+		FriendbotURL:                viper.GetString("friendbot-url"),
+		LogLevel:                    ll,
+		SentryDSN:                   viper.GetString("sentry-dsn"),
+		LogglyToken:                 viper.GetString("loggly-token"),
+		LogglyHost:                  viper.GetString("loggly-host"),
+		TLSCert:                     cert,
+		TLSKey:                      key,
+		Ingest:                      viper.GetBool("ingest"),
+		HistoryRetentionCount:       uint(viper.GetInt("history-retention-count")),
+		StaleThreshold:              uint(viper.GetInt("history-stale-threshold")),
+		SkipCursorUpdate:            viper.GetBool("skip-cursor-update"),
+		HorizonDBMaxOpenConnections: viper.GetInt("horizon-db-max-open-connections"),
+		HorizonDBMaxIdleConnections: viper.GetInt("horizon-db-max-idle-connections"),
```

Comment:
I think this should called "history" instead of "horizon", like everywhere else. wdyt?

Reply:
never mind, "horizon" makes more sense here.